src/plugins/spreadsheet/engine/functions/logical.ts

```
@@ -0,0 +1,287 @@
+        condExpr = condExpr.replace(
+          new RegExp(escapedRef.replace(/[.*+?^${}()|[\]\\]/g, "\\$&"), "g"),
+          String(cellValue),
+        );
+      }
+    }
+
+    // Evaluate the condition
+    let conditionResult = false;
+
+    if (/>=|<=|>|<|==|!=/.test(condExpr)) {
+      conditionResult = eval(condExpr);
```

Biome (2.4.10)

[error] eval() exposes to security risks and performance issues.

(lint/security/noGlobalEval)